src/angular-business/chip/chip-input/chip-input.component.ts

```diff
@@ -166,21 +178,23 @@ export class ChipInputComponent extends SbbChipsMixinBase
   private _focused = false;
 
   inputModel = '';
-  origin = new AutocompleteOriginDirective(this._elementRef);
   selectionModel: SelectionModel<string>;
 
-  /** Whether filling out the chip-input is required in the form. */
-  private _required = false;
+  /** Emits when the state of the option changes and any parents have to be notified. */
+  readonly stateChanges = new Subject<void>();
+
+  /**
+   * TODO: Prefix with _
```

Comment:
Wouldn't it be better to already prefix it with "_" and provide a getter to origin?